package.json

```diff
@@ -22,8 +22,7 @@
   "scripts": {
     "build": "gulp",
     "lint": "eslint .",
-    "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
-    "prepublish": "npm run build",
+    "prepublish": "npm run build && node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
```

Comment:
What do you think about doing the `check-dev-engines` check _before_ the `build` (to fail fast)?